aspnetcore/tutorials/min-web-api.md

```diff
@@ -92,23 +92,23 @@ This tutorial creates the following API:
 * In Visual Studio for Mac 2022, select **File** > **New Project...**.
 
 * In the **Choose a template for your new project** dialog:
-  * Select **Web and Console** > **App** > **Empty**.
-  * Select **Continue**.
+  * Select **Web and Console** > **App** > **Empty**
```

Comment:
Why add periods in lines 67-69 but in the same kind of text remove periods in lines 95-96, 104, 111?